engines/osxnative/user_test.go

```diff
@@ -0,0 +1,58 @@
+package osxnative
+
+import (
+	"os"
+	osuser "os/user"
+	"testing"
+)
+
+func TestUser(t *testing.T) {
```

Comment:
Tests should probably be added for the engine that tests running two sandboxes at the same time attempting to access eachothers environment.